README.md

```diff
@@ -36,6 +37,29 @@ or you can choose configuring AWS discovery using [Discovery SPI](http://docs.ha
 Starting with hazelcast-aws-1.2, it's strongly recommended to use Discovery SPI implementation as
 the former one will be deprecated.
 
+### Zone Aware Support
+
+Supported Hazelcast client version is `>= 3.8.6`
+
+As discovery services, these plugins put zone information to the Hazelcast member attributes map during the discovery process. 
```

Comment:
What do you refer to when you use `those plugins`? This sentence is not very clear to me.

Reply:
I think it should be as follows:
"As a discovery service, Hazelcast AWS plugin put the zone information into the Hazelcast's member attributes map during the discovery process. Please see the [Defining Member Attributes section](http://docs.hazelcast.org/docs/latest/manual/html-single/index.html#defining-member-attributes) to learn about the member attributes."